locales/es.yml

```diff
@@ -62,16 +63,16 @@ es:
     reactions: 'reacciones'
     story: 'Artículo'
   flagged_usernames:
-    notify_approved: '{0} approved username {1}'
-    notify_rejected: '{0} rejected username {1}'
-    notify_flagged: '{0} reported username {1}'
-    notify_changed: 'user {0} changed their username to {1}'
+    notify_approved: '{0} nombre de usuario aprobado {1}'
+    notify_rejected: '{0} nombre de usuario rechazado {1}'
+    notify_flagged: '{0} nombre de usuario reportado {1}'
```

Comment:
These -ed words are verbs not nouns, the sentence in English with more context would be:

"Kim Gardner approved username myname124"

So I think it should be:

- notify_approved: '{0} ha aprovado nombre de usuario {1}'
- notify_rejected: '{0} ha rechazado nombre de usuario {1}'
- notify_flagged: '{0} ha reportado nombre de usuario {1}'

What do you think??

Reply:
this is correct @kgardnr !